frontend/public/components/_dropdown.scss

```diff
@@ -238,6 +238,8 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
   border-width: 0 !important; // TEMP fix until https://github.com/patternfly/patternfly-next/issues/2019 is fixed upstream
   list-style: none;
   -webkit-overflow-scrolling: touch;
+  max-height: 60vh;
```

Comment:
I'd make this a variable since it's used for other dropdown heights. That way we always use the same max-height.

Reply:
I think we probably want to scope this fix to the places we know it is needed.  We've had issues in the past sizing all dropdowns generically and ended up creating new bugs in untested contexts.

[vh units are problematic on mobile.](https://css-tricks.com/the-trick-to-viewport-units-on-mobile/)  I suggest we avoid using them here.  We've had a number of bugs in the past trying to size dropdown menus in this way.  @sg00dwin , do you remember where we landed on an approach that worked in all contexts?

Reply:
@sg00dwin Could you help point me some history or PRs? Thank you

Reply:
@yapei, we talked about this at scrum today.  `60vh` is probably ok to use in this instance, but please scope the fix to this one dropdown.  We don't want this fix applied to all dropdowns.

Reply:
PR to @yapei branch with a way to scope this fix to just the monitoring dashboard dropdowns.
https://github.com/yapei/console/pull/1

Reply:
Thank you so much @rhamilto